### PR TITLE
security #7

### PR DESCRIPTION
**Pull request by flexsurfer**
https://github.com/status-im/security-reports/issues/7

### CODE REVIEW

**Review by vitvly — COMMENTED** [review]
I don't have access to security-reports repo. Why do we need to delay calls to navigation-state-changed?

**Comment by flexsurfer** [comment]
to fix #7 ) 
`setInterval(location.assign("https://www.hackerone.com") 5);`

**Comment by vitvly** [comment]
@flexsurfer oh, ok :)

**Review by vitvly — APPROVED** [review]

File: src/status_im/ui/screens/browser/views.cljs [hunk 1]
```diff
@@ -24,6 +24,12 @@
    [status-im.utils.slurp :refer [slurp]]
    [status-im.utils.views :as views]))
 
+(def timeout (atom {}))
+
+(defn debounce [event]
+  (when @timeout (js/clearTimeout @timeout))
+  (reset! timeout (js/setTimeout #(re-frame/dispatch event) 500)))
```
Comment (yenda) [inline src/status_im/ui/screens/browser/views.cljs:31]: so it turns every timeout into a 500ms timeout?

**Comment by Serhy** [comment]
@flexsurfer I can't reproduce issue (https://github.com/status-im/security-reports/issues/7#issuecomment-506399457) anymore with iOS 12, Android 6.0.1, Android 8.1, Android 9
Can use DApps as before (buy and sell cryptokitty, transactions in SimpleDapp)

**Pull request merged**